src/data/faq.js

```diff
@@ -254,37 +254,31 @@ export default [
         answer: (
           <>
             <p>
-              The idea is to calculate the probability of not getting Fusion
-              stones in a book — other calculations are just small variations of
-              this one. To do that, we lay down all the different “drawing
-              sequences”; 64 in a Mythic book (2 different rarities to the power
-              of 6 draws), 4096 is a Classic book (4 different rarities to the
-              power of 6 draws). For example, ‘EEELEL’ (Epic, Epic, Epic,
-              Legendary, Epic, Legendary) is a valid and likely sequence for a
-              Mythic book. So at the core, it is a function that takes a drawing
-              sequence and returns the probability of getting this sequence, and
-              no fusion stones out of a book.
+              The book calculator calculates the probability of not getting
```

Comment:
Thank you for clarifying the explanation, that’s very much appreciated! Suggestion for next time: do related changes together instead of all at once. So for instance, news links in one pull-request, and books explanation in another. This is when branches come in handy: you can separate your work in independent streams. 

I can help you creating branches in the future. :)